src/components/VNavigationDrawer/VNavigationDrawer.js

```diff
@@ -328,10 +328,12 @@ export default {
         transitionend: e => {
           this.$emit('transitionend', e)
 
-          // IE11 does not support new Event('resize')
-          const resizeEvent = document.createEvent('UIEvents')
-          resizeEvent.initUIEvent('resize', true, false, window, 0)
-          window.dispatchEvent(resizeEvent)
+          if(e.srcElement.localName === 'aside') {
```

Comment:
There could be other `aside` element in the drawer, maybe `e.srcElement === this.$el`? Didn't check if it works though

Reply:
And use `event.target`, `srcElement` is a non-standard property from IE.

Reply:
from the vue guide
https://vuejs.org/v2/guide/render-function.html#Event-amp-Key-Modifiers
if (e.target !== e.currentTarget) return